scripts/pages/view.py

```diff
@@ -291,6 +295,13 @@ def displayNotes(hide=False):
         print "</table></div></form>"
 
     # And the 'new' section, for adding enclosures
+    # Note the textareas are split into "Edit" and a hidden "Send" to replace
```

Comment:
Could you clarify what exactly the Edit and Send boxes are each responsible for here? For example, something like "the Edit box performs xyz, and should contain the text with/without unicode/newlines characters" (but with the correct information).